pangeo_forge_orchestrator/cli.py

```diff
@@ -0,0 +1,65 @@
+import ast
+import os
+
+import typer
+
+from .client import Client
+
+cli = typer.Typer()
+db = typer.Typer()
+client = Client(base_url=os.environ["PANGEO_FORGE_DATABASE_URL"])
```

Comment:
> Is that the desired behavior?

Yes, the idea is that users need to provide a `"PANGEO_FORGE_DATABASE_URL"` to use the cli. By "database" here, we actually mean the API endpoint for interacting with the database, but I'm using this name for consistency because 

```zsh
$ pangeo-forge database
```
is the cli module name (not `pangeo-forge api`). Which is all to say, if this is unset we *do* want to raise a `KeyError`.

Would

```python
try:
    client = Client(base_url=os.environ["PANGEO_FORGE_DATABASE_URL"])
except KeyError as e:
    raise KeyError(
         "Environment variable 'PANGEO_FORGE_DATABASE_URL' not set. "
         "Set this variable to a local or remote URL serving an instance of "
         "`pangeo_forge_orchestrator.api:api`."
    ) from e
```
be preferable?

Reply:
It's fine as is for now.

Reply:
Per our discussion today, I now understand that the notion of a required env variable is somewhat of an anti-pattern. It is more conventional to have a default argument which can be overridden by an env variable if desired.

Because we don't yet have a deployed API instance, we don't yet have a default URL to supply here. We are therefore going to leave this as-is for now, and revise post-deployment.